src/NodeAFK.js

```diff
@@ -0,0 +1,55 @@
+const Listener = require('./Listener');
+
+class NodeAFK {
+  constructor() {
+    this.listeners = {};
+    this.nextListenerId = 1;
+  }
+
+  getAllListeners() {
+    return this.listeners;
+  }
+
+  // Add a listener to check when the user is away
```

Comment:
Would this read better: `Add a listener to report on the user's activity status` as the callback is not specifically concerned if the user is away